test/dummy/db/schema.rb

```diff
@@ -96,31 +127,38 @@
 
   add_index "post_versions", ["item_type", "item_id"], name: "index_post_versions_on_item_type_and_item_id"
 
-  create_table "post_with_statuses", force: true do |t|
+  create_table "post_with_statuses", force: :cascade do |t|
     t.integer "status"
   end
 
-  create_table "posts", force: true do |t|
+  create_table "posts", force: :cascade do |t|
     t.string "title"
     t.string "content"
   end
 
-  create_table "songs", force: true do |t|
+  create_table "skippers", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "another_timestamp"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  create_table "songs", force: :cascade do |t|
     t.integer "length"
   end
 
-  create_table "things", force: true do |t|
+  create_table "things", force: :cascade do |t|
     t.string "name"
   end
 
-  create_table "translations", force: true do |t|
+  create_table "translations", force: :cascade do |t|
```

Comment:
Why did the `force` option change from `true` to `:cascade` on every table? Is that the new recommended value, or is it something you did intentionally? From the docs:

```
:force
Set to true to drop the table before creating it. Set to :cascade to drop dependent objects as well. Defaults to false.
```

It seems like with `force: true` the order of table creation wouldn't matter, but with `:cascade` it would, so `true` seems preferable to me.